### DISCUSSION TITLE
what's the difference from original repo?

### DISCUSSION THREAD

**Discussion by MithrilMan** [idiap/coqui-ai-TTS#61] [Q&A] [answered]
Hello,
what's the difference between this fork and the original repo and what's your "end goal"?
I'm interested in a solution I can deploy on docker locally and use as a personal assistant voice and should act realtime (supporting italian voice)

Does this project fits the need (or will it ever?)
Thanks and godspeed!

**Comment by eginhard** [comment, marked as answer]
There shouldn't be any major differences, so far we added mostly bug fixes and a few small features. You can continue using this repo in the same way. For a detailed list of changes, see https://github.com/idiap/coqui-ai-TTS/releases

For development goals, also see #23. Italian is supported by multiple Coqui models.

**Reply by MithrilMan** [reply]
Thanks I will take a look
